plugins/catalog-react/src/alpha/blueprints/extensionData.tsx

```diff
@@ -32,3 +32,19 @@ export const entityFilterExpressionDataRef =
   createExtensionDataRef<string>().with({
     id: 'catalog.entity-filter-expression',
   });
+
+/**
+ * @alpha
+ * Default entity content groups.
+ */
+export const defaultEntityContentGroups = {
```

Comment:
Open to suggestions on what the default groups should be

Reply:
Yep I think this is a good starting point but might need another pass before this hits mainline